src/pages/index.js

```diff
@@ -8,38 +8,66 @@ import Section from "../components/Section.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import UserInfo from "../components/UserInfo.js";
+import Api from "../components/api";
+import PopupWithConfirmation from "../components/PopupWithConfirmation";
 
 const addCardForm = document.querySelector("#add-card-form");
 const editProfileForm = document.querySelector("#edit-profile-form");
+const editAvatarForm = document.querySelector("#edit-avatar-form");
+
 const addNewCardButton = document.querySelector(".profile__add-button");
 const editProfileButton = document.querySelector(".profile__edit-button");
+const editAvatarButton = document.querySelector(".profile__image-edit-button");
+// const cardDeleteButton = document.querySelector(".card__delete-button");
 
 // create instances of classes
 
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "538c4c21-92b1-4aec-a8cb-8245624705df",
+    "Content-Type": "application/json",
+  },
+});
+let cardSection;
+
+api.getInitialCards().then((res) => {
+  cardSection = new Section(
+    {
+      items: res,
+      renderer: (data) => {
+        const cardEl = createCard(data);
+        cardSection.addItem(cardEl);
+      },
+    },
+    selectors.cardSection
+  );
+  cardSection.renderItems();
+});
+
+api.getUserInfo().then((res) => {
+  userInfo.setAvatar(res.avatar);
+  userInfo.setUserInfo(res);
+  // userInfo.setUserInfo(res.about);
+});
+
 const userInfo = new UserInfo({
   nameSelector: ".profile__title",
   jobSelector: ".profile__description",
+  userAvatar: ".profile__image",
 });
 
 const addCardFormValidator = new FormValidator(config, addCardForm);
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object `formValidators`.  And then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
const formValidators = {}

// enable validation
const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // here you get the name of the form
    const formName = formElement.getAttribute('name')

   // here you store a validator by the `name` of the form
    formValidators[formName] = validator;
   validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```